userspace/libsinsp/sinsp.h

```diff
@@ -945,6 +945,33 @@ class SINSP_PUBLIC sinsp : public capture_stats_source, public wmi_handle_source
 
 	uint64_t get_lastevent_ts() const { return m_lastevent_ts; }
 
+	/*!
+	  \brief Simple set of syscalls that enforce minimum sinsp state set,
+	  plus some valuable syscalls.
+	  It has same effect of old `simple_consumer` mode.
```

Comment:
here I would add if it enforces the minimum `sinsp` state, looking at the implementation is clear but for a user maybe not